### PR TITLE
[Feature] Add disable exchange rate variable

### PR DESCRIPTION
**Pull request by fivetran-avinash**
## PR Overview
**This PR will address the following Issue/Feature:** 
Issues from transform package 
https://github.com/fivetran/dbt_netsuite/issues/70

**This PR will result in the following new package version:** 0.4.0
 

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:** Introduces variable `netsuite2__using_exchange_rate` to allow users who don't utilize exchange rates in Netsuite2 the ability to disable that functionality.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [x] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

Models run successfully whether vars are configure to true or false.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [NA] DECISIONLOG updates have been updated (if applicable)
- [NA] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
💱

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,8 +113,10 @@ It's possible that your Netsuite connector does not sync every table that this p
 ```yml
 vars:
     netsuite2__multibook_accounting_enabled: false # True by default. Disable `accountingbooksubsidiary` and `accountingbook` if you are not using the Multi-Book Accounting feature
+    netsuite2__using_exchange_rate: false #True by default. Disable `exchange_rate` if you don't utilize exchange rates. Also set this variable to `false` in the `dbt_netsuite` package if you are utilizing it.
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply README.md:116]: also the vars in this snippet are scoped globally, so maybe we don't need this note?

**Review by fivetran-jamie — APPROVED** [review]
code looks good, just some docs comments, so i shall approve

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply README.md]: Updated!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
-# dbt_netsuite_source v0.UPDATE.UPDATE
+# dbt_netsuite_source v0.6.3
+[PR #35](https://github.com/fivetran/dbt_netsuite_source/pull/35) applies the following changes:
 
- ## Under the Hood:
+## 🎉 Feature Updates 🎉
+- Introduces variable `netsuite2__using_exchange_rate` to allow users who don't utilize exchange rates in Netsuite2 the ability to disable that functionality within your `dbt_project.yml`, and return only the unconverted amount as the final converted amount.
+- **Note**: If you are using the `dbt_netsuite` package, you will also want to set `netsuite2__using_exchange_rate` to false there as well to avoid compilation errors.
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md]: Updated!

**Pull request merged**